cli/valet.php

```diff
@@ -190,10 +190,13 @@
      * Unlink a link from the Valet links directory.
      */
     $app->command('unlink [name]', function ($name) {
-        Site::unlink($name = $name ?: basename(getcwd()));
-
-        info('The ['.$name.'] symbolic link has been removed.');
-    })->descriptions('Remove the specified Valet link');
+        $result = Site::unlink($name = $name ?: basename(getcwd()));
+        if (!$result) {
```

Comment:
```suggestion
        if (!Site::unlink($name = $name ?: basename(getcwd()))) {
```

$result does not seem to be reused, so the boolean returned by the function can be used immediately.